fastlane/Pluginfile

```diff
@@ -0,0 +1,5 @@
+# Autogenerated by fastlane
+#
+# Ensure this file is checked in to source control!
+
+gem 'fastlane-plugin-firebase_app_distribution'
```

Comment:
You can remove this plugin since it is unused.

Reply:
Fixed.